### PR TITLE
fix Map casting in Assert.same

### PR DESCRIPTION
**Pull request by profelis**
Fix incorrect casting to Map. Btw, maybe you can use this

https://github.com/fponticelli/thx.core/blob/master/src/thx/core/Dynamics.hx#L10

### CODE REVIEW

**Comment by fponticelli** [comment]
Thank you sir! I will publish a new release ASAP.

**Comment by fponticelli** [comment]
About thx.core I am planning on writing utest v.2 using other libraries ... it will just take some time ;)

**Comment by profelis** [comment]
Thank you, too!

File: src/utest/Assert.hx [hunk 1]
```diff
@@ -284,10 +284,13 @@ unless you know what you are doing.
         }
 
         // hash, inthash
-        if(Std.is(expected, haxe.ds.StringMap) || Std.is(expected, haxe.ds.IntMap)) {
+        if (Std.is(expected, Map.IMap)) {
```
Comment (ponticelli) [inline src/utest/Assert.hx:287]: apparently IMap has been recently deprecated so I believe we will have to rollback to individual tests :(

**Comment by fponticelli** [comment]
using haxe.Constraints.IMap should do the trick

**Pull request merged**